build/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0.400-alpine3.13 AS build
+FROM mcr.microsoft.com/dotnet/sdk:5.0.402-alpine3.13 AS build
```

Comment:
The runtime should also be updated to 5.0.11 for this SDK

Reply:
https://github.com/microsoft/fhir-server/blob/main/samples/templates/default-azuredeploy.json#L336 should also stay in sync